com.unity.cinemachine/Documentation~/samples-tutorials.md

```diff
@@ -2,15 +2,48 @@
+| **Fly around** | First-person flight simulator. | <ul> <li>Set up a fly-around camera with basic height and speed controls. |
+| **FreeLook Deoccluder** | Third-person free look with obstacle handling. | <ul> <li>Set up a free look camera that handles occlusion by walls to keep the player in view.</li> <li>Set up the scene and the camera to consider certain objects as transparent and ignore them in the occlusion evaluation.</li> </ul> |
+| **FreeLook on Spherical Surface** | Third-person free look with local up direction. | <ul> <li>Set up a free look camera to follow a player that walks on the surface of a sphere.</li> <li>Set up the camera to follow the character either lazily or actively.</li> </ul> |
+| **Impulse Wave** | Third-person follow and impulse wave effect. | <ul> <li>Set up the scene and the camera to make them react to impulse waves.</li> <li>Invoke an impulse from a fixed epicenter.</li> <li>Trigger an impulse when the player jumps.</li> </ul> |
+| **Lock-on Target** | Third-person follow with boss area. | <ul> <li>Set up a simple third-person free look camera to look at the player and rotate around it with the mouse.</li> <li>Set up a camera that looks at the player and locks on a boss character when the player enters a trigger zone.</li> </ul>  |
+| **Mixing Camera** | Car racing game with stunt. | <ul> <li>Set up a camera group that continuously blends multiple cameras to reflect the car speed.</li> <li>Set up a fixed camera that activates for a cut-in when the car enters a two-ramp stunt zone, to frame the car jump from the side.</li> </ul> |
+| **Running Race** | Character race game (simulation). | <ul> <li>Set up a group of cameras that each follow a different player.</li> <li>Get the cameras automatically switch to always have the leader of the race centered on screen.</li> <li>Set up an on-demand camera that frames all runners while looking at the sun.</li> <li>Emulate players that move along predefined paths.</li> </ul>  |
+| **Split Screen Car** | Local multiplayer car racing game. | <ul> <li>Display two racing cars in a split screen configuration.</li> </ul> |
+| **Third Person With Aim Mode** | Third-person shooter. | <ul> <li>Set up a simple free look camera that follows the player as it moves, jumps, and sprints.</li> <li>Add noise to the camera to emulate a handheld effect.</li> <li>Set up an on-demand camera that allows the player to aim and fire projectiles precisely with a crosshair despite the handheld effect.</li> </ul> |
+| **Third Person With Roadie Run** | Third-person shooter. | <ul> <li>Set up a simple free look camera that follows the player as it moves, jumps, aims, and fires projectiles with a crosshair.</li> <li>Set up a camera recoil effect when the player fires projectiles.</li> <li>Set up a special roadie run camera that automatically activates when the player sprints, with no crosshair and more noise.</li> </ul> |
```

Comment:
Also an important sample using the ThirdPerson paradigm which is distinct from the FreeLook paradigm.  This key difference is a major point of confusion in the wild and we need to pay special attention to it in the doc

Reply:
Exactly! Definitely, I want to make it crystal clear in the docs. I'll prepare another round on this PR.